File: backend/HereHear/src/main/java/com/ssafy/herehear/achievement/dto/AchievementDto.java
package com.ssafy.herehear.achievement.dto;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
public class AchievementDto {
    private Long achievementId;
    private String mission;
    private BadgeCodeDto badge;
    private TitleCodeDto title;
}

File: wearos/HereHear/app/src/main/java/com/ssafy/herehear/presentation/data/AchievementDto.kt
package com.ssafy.herehear.presentation.data

data class AchievementDto(
    val achievementId: Int = 0,
    val mission: String = "",
    val badge: BadgeDto = BadgeDto(),
    val title: TitleDto = TitleDto()
)

data class BadgeDto(
    val badgeCode: Int = 0,
    val badgeName: String = "",
    val badgeImg: String = ""
)

data class TitleDto(
    val titleCode: Int = 0,
    val titleName: String = ""
)